node/coinstacks/solana/api/src/controller.ts

```diff
@@ -73,9 +79,67 @@ export class Solana implements BaseAPI {
   @Response<ValidationError>(422, 'Validation Error')
   @Response<InternalServerError>(500, 'Internal Server Error')
   @Get('account/{pubkey}/txs')
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  async getTxHistory(@Path() pubkey: string, @Query() _cursor?: string, @Query() _pageSize = 10): Promise<TxHistory> {
-    return { pubkey } as TxHistory
+  async getTxHistory(
+    @Path() pubkey: string,
+    @Query() lastSignature?: string,
```

Comment:
preference to still use the cursor pattern. it can contain `lastSignature` as a means of paginating against the helius endpoint, but obscures things and allows other metadata to be passed if needed.

Reply:
Changed in [ccc3ac6](https://github.com/shapeshift/unchained/pull/1052/commits/ccc3ac6c3a7e216025c7dfa0a40266543f69a764)